tools/dds/dds-server/lrs-device-manager.cpp

```diff
@@ -0,0 +1,119 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#include <algorithm>
+#include <iostream>
+#include "lrs-device-manager.h"
+
+using namespace tools;
+
+class lrs_device_manager::lrs_sensor_streamer
+{
+public:
+    lrs_sensor_streamer() = default;
```

Comment:
If this happens, the sensor won't ever be opened and started... in use?

Reply:
Will be removed